frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderLeft/DigdirLogoLink/DigdirLogoLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classes from './DigdirLogoLink.module.css';
+import { DigdirLogo } from './DigdirLogo';
+import { Paragraph } from '@digdir/design-system-react';
+import { StudioPageHeaderButton } from '../../StudioPageHeaderButton';
+import { useStudioPageHeaderContext } from '../../context';
+
+export type DigdirLogoLinkProps = {
+  title?: string;
+};
+
+export const DigdirLogoLink = ({ title }: DigdirLogoLinkProps): React.ReactElement => {
+  const { variant } = useStudioPageHeaderContext();
+  const color = variant === 'regular' ? 'dark' : 'light';
+
+  return (
+    <div className={classes.wrapper}>
+      <StudioPageHeaderButton asChild color={color} variant={variant}>
+        <a href='/'>
```

Comment:
Can we use Link from the design system (studio-components) here? 😊

Reply:
By doing that, the style of the StudioPageHeaderButton is overridden by the designsystem component. As this link is just a simple link with a `href` to `/` I thought it was a quicker and better solution to keep it as it is now 😄